### DISCUSSION TITLE
Tree-sitter `queries` directory not in `runtimepath`?

### DISCUSSION THREAD

**Discussion by juangiordana** [nix-community/kickstart-nix.nvim#53] [Q&A] [answered]
Hello,

I've been trying to add custom Tree-sitter directives by creating a `queries` directory to mimic what this plugin is doing
https://github.com/ngynkvn/gotmpl.nvim but I couldn't make it work.

The idea behind the plugin is to support multiple languages for a  file type, i.e. `html` within `gotmpl` files.

Could this be because the `queries` directory is not added to the `runtimepath`?

I've been messing around the whole day without luck. Any ideas?

https://neovim.io/doc/user/treesitter.html#treesitter-query

Thanks

**Comment by mrcjkb** [comment, marked as answer]
Hey :wave: 

Try adding

```scheme
;; extends
```

to the top of your query file.

**Reply by juangiordana** [reply]
That made it. Thanks a lot!

Leaving these here as reference to anyone interested:
```
:help treesitter-language-injections
:help treesitter-query-modeline-extends
```
